components/src/dynamo/vllm/tests/test_vllm_omni_handler.py

```
@@ -161,6 +160,24 @@ def test_finish_reason_included(self):
         assert chunk["choices"][0]["finish_reason"] == "stop"
         assert "usage" in chunk
 
+    def test_finish_reason_abort_normalized(self):
+        """Abort finish reason is normalized to 'cancelled'."""
+        handler = _make_handler()
+        handler._build_completion_usage = lambda ro: {
```

Ruff (0.15.2)

[warning] Unused lambda argument: `ro`

(ARG005)